R/grasslandInventory.R

```diff
@@ -0,0 +1,18 @@
+#' A dataset indicating sampling years for the grasslandData example dataset
```

Comment:
I read this data.frame into R and the way it is structured is not intuitive to me. Why does it need to be in this format?

Reply:
The previous version of the tracking scripts require this format, so I've sort of stuck with it just because that's how the data are stored right now. 
I need a record of when each quadrat was sampled, so that we can differentiate between a case when a plant actually goes dormant for one year, or if the quadrat just wasn't sampled in that year. Do you have another format idea that would make more sense?

Reply:
@aestears 

We can talk about this more on Tuesday but would something like this be more intuitive to the users:
year | quadrat
2010 | 1
2010 | 2
2010 | 3
etc.